test/registries.ts

```diff
@@ -277,4 +277,28 @@ describe('Registries', () => {
       'Public endpoint can not be empty',
     );
   });
+
+  it("can't register a seeker using the same seeker ID", async() => {
```

Comment:
I think we _can_ register a seeker using the same seeker ID, because there is no error in the code. Should we say 

```suggestion
  it("registered node seeker id will be reset if a new node registers with the same seeker id", async() => {
```

Reply:
Yea that is a better description I think